evmrpc/tx.go

```diff
@@ -84,6 +84,14 @@ func (t *TransactionAPI) GetTransactionByHash(ctx context.Context, hash common.H
 	return t.GetTransactionByBlockNumberAndIndex(ctx, rpc.BlockNumber(receipt.BlockNumber), hexutil.Uint(receipt.TransactionIndex)), nil
 }
 
+func (t *TransactionAPI) GetTransactionErrorByHash(ctx context.Context, hash common.Hash) (string, error) {
```

Comment:
this seems nonstandard, is it needed to surface errors to metamask users?

Reply:
it is more for developers' debugging usage